--- xkeyboard-config-2.4.orig/keycodes/sun	2011-02-19 15:32:36.000000000 +0000
+++ xkeyboard-config-2.4/keycodes/sun	2012-04-06 00:41:05.431314792 +0100
@@ -550,46 +550,50 @@
 xkb_keycodes "type6_usb" {
     include "xfree86"
 
-    <STOP> = 232;
-    <AGAI> = 133;
-    <PROP> = 134;
-    <UNDO> = 135;
-    <FRNT> = 140;
-    <COPY> = 248;
-    <OPEN> = 191;
-    <PAST> = 192;
-    <FIND> = 122;
-    <CUT>  = 188;
+//    <STOP> = 232;
+//    <AGAI> = 133;
+//    <PROP> = 134;
+//    <UNDO> = 135;
+//    <FRNT> = 140;
+//    <COPY> = 248;
+//    <OPEN> = 191;
+//    <PAST> = 192;
+//    <FIND> = 122;
+//    <CUT>  = 188;
 
-    <HELP> = 245;
+ //   <HELP> = 245;
 
     // The blank has keycode 239 on my type6 serial kb, but 134 on
     // my type6 usb keyboard (same as <PROP>)
-    <BLNK> = 239;
+//    <BLNK> = 239;
 
     // AltGr + PrScr actually sends a different keycode
-    <SYRQ> = 92;
+//    <SYRQ> = 92;
 
-    <MUTE> = 160;
-    <VOL-> = 174;
-    <VOL+> = 176;
-    <POWR> = 222;
+//    <MUTE> = 160;
+//    <VOL-> = 174;
+//    <VOL+> = 176;
+//    <POWR> = 222;
+
+      <I150> = 155;
+      <I160> = 160;
+      <I235> = 235;
 
     indicator 4 = "Compose";
 };
 
 xkb_keycodes "type6tuv_usb" {
     include "sun(type6_usb)"
-    <LSGT> = 94;
+//    <LSGT> = 94;
 //    <BKSL> = 51;
 };
 
 xkb_keycodes "type6_jp_usb" {
     include "sun(type6_usb)"
     alias <UNDR> = <KPDC>;  // 123
-    alias <EXEC> = <I0B>;   // 139 
-    alias <KANJ> = <I0A>;   // 138
-    alias <HENK> = <XFER>;  // 129
+    alias <EXEC> = <NFER>;  // 131
+    alias <KANJ> = <XFER>;  // 129
+    alias <HENK> = <HKTG>;  // 208
     indicator 5 = "Kana";
 };
 
