--- xkeyboard-config-2.4.orig/symbols/sun_vndr/ara	2011-02-19 15:32:37.000000000 +0000
+++ xkeyboard-config-2.4/symbols/sun_vndr/ara	2012-04-06 00:43:17.832163209 +0100
@@ -111,6 +111,49 @@
 
 
 partial alphanumeric_keys
+	xkb_symbols "qwerty" {
+	include "ara(qwerty)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "azerty" {
+	include "ara(azerty)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "digits" {
+	include "ara(digits)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "qwerty_digits" {
+	include "ara(qwerty_digits)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "azerty_digits" {
+	include "ara(azerty_digits)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "buckwalter" {
+	include "ara(buckwalter)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "olpc" {
+	include "ara(olpc)"
+};
+
+
+partial default alphanumeric_keys
 	xkb_symbols "basic" {
 	include "sun_vndr/ara(type6)"
 };
+
