--- xkeyboard-config-2.4.orig/symbols/sun_vndr/cz	2011-02-19 15:32:37.000000000 +0000
+++ xkeyboard-config-2.4/symbols/sun_vndr/cz	2012-04-06 00:46:05.256283030 +0100
@@ -100,6 +100,28 @@
 };
 
 partial alphanumeric_keys
+	xkb_symbols "bksl" {
+	include "cz(bksl)"
+};
+
+partial alphanumeric_keys 
+	xkb_symbols "qwerty" {
+	include "cz(qwerty)"
+};
+
+partial alphanumeric_keys
+	xkb_symbols "qwerty_bksl" {
+	include "cz(qwerty_bksl)"
+};
+
+partial alphanumeric_keys alternate_group
+	xkb_symbols "ucw" {
+	include "cz(ucw)"
+};
+
+
+partial default alphanumeric_keys
 	xkb_symbols "basic" {
 	include "sun_vndr/cz(type6)"
 };
+
