--- xkeyboard-config-2.4.orig/symbols/sun_vndr/dk	2011-02-19 15:32:37.000000000 +0000
+++ xkeyboard-config-2.4/symbols/sun_vndr/dk	2012-04-06 00:46:46.630121242 +0100
@@ -142,6 +142,31 @@
 
 
 partial alphanumeric_keys
+	xkb_symbols "nodeadkeys" {
+	include "dk(nodeadkeys)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "mac" {
+	include "dk(mac)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "mac_nodeadkeys" {
+	include "dk(mac_nodeadkeys)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "dvorak" {
+	include "dk(dvorak)"
+};
+
+
+partial default alphanumeric_keys
 	xkb_symbols "basic" {
 	include "sun_vndr/dk(type6)"
 };
+
