--- xkeyboard-config-2.4.orig/symbols/sun_vndr/ch	2011-02-19 15:32:37.000000000 +0000
+++ xkeyboard-config-2.4/symbols/sun_vndr/ch	2012-04-06 00:45:55.991164356 +0100
@@ -128,6 +128,67 @@
 
 
 partial alphanumeric_keys
+	xkb_symbols "de" {
+	include "ch(de)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "de_Sundeadkeys" {
+	include "ch(de_Sundeadkeys)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "de_sundeadkeys" {
+	include "ch(de_sundeadkeys)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "de_nodeadkeys" {
+	include "ch(de_nodeadkeys)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "fr" {
+	include "ch(fr)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "fr_Sundeadkeys" {
+	include "ch(fr_Sundeadkeys)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "fr_sundeadkeys" {
+	include "ch(fr_sundeadkeys)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "fr_nodeadkeys" {
+	include "ch(fr_nodeadkeys)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "de_mac" {
+	include "ch(de_mac)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "fr_mac" {
+	include "ch(fr_mac)"
+};
+
+
+partial default alphanumeric_keys
 	xkb_symbols "basic" {
 	include "sun_vndr/ch(de_type6)"
 };
+
