--- xkeyboard-config-2.4.orig/symbols/sun_vndr/ee	2011-02-19 15:32:37.000000000 +0000
+++ xkeyboard-config-2.4/symbols/sun_vndr/ee	2012-04-06 00:47:23.670944512 +0100
@@ -112,6 +112,23 @@
 };
 
 partial alphanumeric_keys
+	xkb_symbols "nodeadkeys" {
+	include "ee(nodeadkeys)"
+};
+
+partial alphanumeric_keys
+	xkb_symbols "dvorak" {
+	include "ee(dvorak)"
+};
+
+partial alphanumeric_keys
+	xkb_symbols "us" {
+	include "ee(us)"
+};
+
+
+partial default alphanumeric_keys
 	xkb_symbols "basic" {
 	include "sun_vndr/ee(type6)"
 };
+
