--- xkeyboard-config-2.4.orig/symbols/sun_vndr/de	2011-02-19 15:32:37.000000000 +0000
+++ xkeyboard-config-2.4/symbols/sun_vndr/de	2012-04-06 00:46:30.951306572 +0100
@@ -133,8 +133,92 @@
 
 };
 
+partial alphanumeric_keys
+	xkb_symbols "nodeadkeys" {
+	include "de(nodeadkeys)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "deadgraveacute" {
+	include "de(deadgraveacute)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "deadacute" {
+	include "de(deadacute)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "ro" {
+	include "de(ro)"
+};
+
 
 partial alphanumeric_keys
+	xkb_symbols "ro_nodeadkeys" {
+	include "de(ro_nodeadkeys)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "dvorak" {
+	include "de(dvorak)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "Sundeadkeys" {
+	include "de(Sundeadkeys)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "sundeadkeys" {
+	include "de(sundeadkeys)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "neo" {
+	include "de(neo)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "mac" {
+	include "de(mac)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "mac_nodeadkeys" {
+	include "de(mac_nodeadkeys)"
+};
+
+
+//partial alphanumeric_keys
+//	xkb_symbols "dsb"
+//	include "de(dsb)"
+//};
+
+
+//partial alphanumeric_keys
+//	xkb_symbols "dsb_qwertz"
+//	include "de(dsb_qwertz)"
+//};
+
+
+partial alphanumeric_keys
+	xkb_symbols "qwerty" {
+	include "de(qwerty)"
+};
+
+
+partial default alphanumeric_keys
 	xkb_symbols "basic" {
 	include "sun_vndr/de(type6)"
 };
+
