--- xkeyboard-config-2.4.orig/symbols/sun_vndr/fr	2011-02-19 15:32:37.000000000 +0000
+++ xkeyboard-config-2.4/symbols/sun_vndr/fr	2012-04-06 00:48:02.533906805 +0100
@@ -86,11 +86,124 @@
     key <AB10>	{ [          exclam,    section                  ] };
 
 	include "level3(ralt_switch)"
+};
+
+partial alphanumeric_keys
+	xkb_symbols "Sundeadkeys" {
+	include "fr(Sundeadkeys)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "sundeadkeys" {
+	include "fr(sundeadkeys)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "nodeadkeys" {
+	include "fr(nodeadkeys)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "oss" {
+	include "fr(oss)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "oss_latin9" {
+	include "fr(oss_latin9)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "oss_Sundeadkeys" {
+	include "fr(oss_Sundeadkeys)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "oss_sundeadkeys" {
+	include "fr(oss_sundeadkeys)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "oss_nodeadkeys" {
+	include "fr(oss_nodeadkeys)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "latin9" {
+	include "fr(latin9)"
+};
+
 
+partial alphanumeric_keys
+	xkb_symbols "latin9_Sundeadkeys" {
+	include "fr(latin9_Sundeadkeys)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "latin9_sundeadkeys" {
+	include "fr(latin9_sundeadkeys)"
 };
 
 
 partial alphanumeric_keys
+	xkb_symbols "latin9_nodeadkeys" {
+	include "fr(latin9_nodeadkeys)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "bepo" {
+	include "fr(bepo)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "bepo_latin9" {
+	include "fr(bepo_latin9)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "dvorak" {
+	include "fr(dvorak)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "bre" {
+	include "fr(bre)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "oci" {
+	include "fr(oci)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "mac" {
+	include "fr(mac)"
+};
+
+
+partial alphanumeric_keys
+	xkb_symbols "geo" {
+	include "fr(geo)"
+};
+
+
+partial default alphanumeric_keys
 	xkb_symbols "basic" {
 	include "sun_vndr/fr(type6)"
 };
+
