--- xkeyboard-config-2.4.orig/symbols/sun_vndr/ro	2011-02-19 15:32:37.000000000 +0000
+++ xkeyboard-config-2.4/symbols/sun_vndr/ro	2012-04-06 00:50:54.982682923 +0100
@@ -108,10 +108,51 @@
    key <AB10> { [          slash,      question,	slash,      question ] };
 
 	include "level3(ralt_switch)"
+};
+
+partial alphanumeric_keys
+	xkb_symbols "cedilla" {
+	include "ro(cedilla)"
+};
+
+partial alphanumeric_keys
+	xkb_symbols "std" {
+	include "ro(std)"
+};
 
+partial alphanumeric_keys
+	xkb_symbols "std_cedilla" {
+	include "ro(std_cedilla)"
 };
 
 partial alphanumeric_keys
+	xkb_symbols "winkeys" {
+	include "ro(winkeys)"
+};
+
+partial
+	xkb_symbols "crh_f" {
+	include "ro(crh_f)"
+};
+
+partial
+	xkb_symbols "crh_alt" {
+	include "ro(crh_alt)"
+};
+
+partial
+	xkb_symbols "crh_dobruca1" {
+	include "ro(crh_dobruca1)"
+};
+
+partial
+	xkb_symbols "crh_dobruca2" {
+	include "ro(crh_dobruca2)"
+};
+
+
+partial default alphanumeric_keys
 	xkb_symbols "basic" {
 	include "sun_vndr/ro(type6)"
 };
+
