--- xkeyboard-config-2.4.orig/symbols/sun_vndr/ua	2011-02-19 15:32:37.000000000 +0000
+++ xkeyboard-config-2.4/symbols/sun_vndr/ua	2012-04-06 00:53:01.190372071 +0100
@@ -108,10 +108,61 @@
     key	<AB10> {	[	    slash,	  question,	slash,	  question	        ]	};
 
 	include "level3(ralt_switch)"
+};
+
+partial alphanumeric_keys 
+	xkb_symbols "legacy" {
+	include "ua(legacy)"
+};
+
+partial alphanumeric_keys
+	xkb_symbols "winkeys" {
+	include "ua(winkeys)"
+};
+
+partial alphanumeric_keys
+	xkb_symbols "typewriter" {
+	include "ua(typewriter)"
+};
+
+partial alphanumeric_keys
+	xkb_symbols "phonetic" {
+	include "ua(phonetic)"
+};
+
+partial alphanumeric_keys
+	xkb_symbols "rstu" {
+	include "ua(rstu)"
+};
 
+partial alphanumeric_keys
+	xkb_symbols "rstu_ru" {
+	include "ua(rstu_ru)"
 };
 
 partial alphanumeric_keys
+	xkb_symbols "homophonic" {
+	include "ua(homophonic)"
+};
+
+partial
+	xkb_symbols "crh" {
+	include "ua(crh)"
+};
+
+partial
+	xkb_symbols "crh_f" {
+	include "ua(crh_f)"
+};
+
+partial
+	xkb_symbols "crh_alt" {
+	include "ua(crh_alt)"
+};
+
+
+partial default alphanumeric_keys
 	xkb_symbols "basic" {
 	include "sun_vndr/ua(type6)"
 };
+
